cmd/svcat/broker/sync_cmd.go

```diff
@@ -20,24 +20,31 @@ import (
 	"fmt"
 
 	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
+	"github.com/kubernetes-incubator/service-catalog/pkg/svcat/service-catalog"
 	"github.com/spf13/cobra"
 )
 
 type syncCmd struct {
-	*command.Context
+	*command.Namespaced
+	*command.Scoped
 	name string
 }
 
 // NewSyncCmd builds a "svcat sync broker" command
 func NewSyncCmd(cxt *command.Context) *cobra.Command {
-	syncCmd := &syncCmd{Context: cxt}
+	syncCmd := &syncCmd{
+		Namespaced: command.NewNamespaced(cxt),
+		Scoped:     command.NewScoped(),
+	}
 	rootCmd := &cobra.Command{
 		Use:     "broker NAME",
 		Short:   "Syncs service catalog for a service broker",
 		Example: command.NormalizeExamples(`svcat sync broker asb`),
 		PreRunE: command.PreRunE(syncCmd),
 		RunE:    command.RunE(syncCmd),
 	}
+	syncCmd.AddScopedFlags(rootCmd.Flags(), true)
```

Comment:
this should be passing false to disable the all scope. All is only applicable for listing (get) but not when targeting a specific resource.

Reply:
yes, this is what I found in the test, I seems forgot to update this field